openbb_terminal/core/sdk/trail_map.csv

```diff
@@ -449,7 +449,7 @@ stocks.fa.info,stocks_fa_yahoo_finance_model.get_info,
 stocks.fa.key,stocks_fa_av_model.get_key_metrics,
 stocks.fa.metrics,stocks_fa_fmp_model.get_key_metrics,
 stocks.fa.mgmt,stocks_fa_business_insider_model.get_management,
-stocks.fa.mktcap,stocks_fa_yahoo_finance_model.get_mktcap,stocks_fa_yahoo_finance_view.display_mktcap
+stocks.fa.mktcap_yf,stocks_fa_yahoo_finance_model.get_mktcap,stocks_fa_yahoo_finance_view.display_mktcap
```

Comment:
We no longer use the convention cmd_source.

Either get rid of yf (wouldn't be mad) or write a function that accepts a source and use it here.

Reply:
Alright, I'll make a new function. In what kind of module should that be included? E.g. it wouldn't fit in fmp_model or yahoofinance_model right?

Reply:
yeah I would throw it in with the others in sdk_helpers.py

Reply:
I mean what about for the Terminal itself where we have model and views?

Reply:
Is fine how it is.  Just the sdk needs a single function

Reply:
Got it

Reply:
I've made `fa.enterprise` and `fa.mktcap` instead.